orchestration/src/main/java/com/sap/ai/sdk/orchestration/Message.java

```diff
@@ -0,0 +1,16 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import javax.annotation.Nonnull;
+
+/** Interface representing convenience wrappers of chat message to the orchestration service. */
+public sealed interface Message permits UserMessage, AssistantMessage, SystemMessage {
```

Comment:
(Major/Discussion/Alignment)

Before release we need to align, whether we use `Orchestration~` as type name prefix or not. Let's discuss in slack.